ceph.spec.in

```diff
@@ -112,6 +112,9 @@
 # this is tracked in https://bugzilla.redhat.com/2152265
 %bcond_with system_arrow
 %endif
+%if 0%{?rhel} <= 8
+%bcond_with lua_packages
+%endif
```

Comment:
if luarocks support is not desired for rhel 8 then the condition should be:
```
%if 0%{?rhel} < 8
%bcond_with lua_packages
%endif
```

no ?

Reply:
we want to support lua packages for rhel 9 and up

Reply:
my bad ... I needed to look up what `%bcond_with` was supposed to do
thanks